### PR TITLE
✨ (frequency): add frequency controls

### PR DESCRIPTION
**Pull request by fakemonster**

### CODE REVIEW

File: src/Frequency.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const frequencyToLogarithmicScale = (frequency, arb = 20) =>
+  Math.log2(frequency/arb)
+
+const Frequency = ({ frequency, onSetFrequencyByField, onSetFrequencyBySlider }) => {
```
Comment (clarkenciel) [inline src/Frequency.jsx]: :nail_care: should be able to do

```javascript
const Frequency = ({ frequency, onSetFrequencyByField, onSetFrequencyBySlider }) => (
  <div>
    ...
  </div>
)

File: src/index.js [hunk 2]
```diff
@@ -80,14 +80,30 @@ const mouse = (state = {}, action) => {
   }
 }
 
+const frequency = (state = 0, action) => {
+  switch (action.type) {
+    case 'SET_FREQUENCY_BY_FIELD':
+      return action.frequency
+    case 'SET_FREQUENCY_BY_SLIDER':
+      return logarithmicScaleToFrequency(action.power)
```
Comment (clarkenciel) [inline src/index.js]: at some point we should move this and `frequencyToLogarithmicScale` to their own module...`math` or something.

File: src/logo.svg [hunk 3]
```diff
@@ -1,7 +0,0 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 841.9 595.3">
```
Comment (clarkenciel) [inline src/logo.svg:1]: good riddance

File: src/index.js [hunk 4]
```diff
@@ -108,6 +124,19 @@ const onMouseUp = e => store.dispatch({
...
+  type: 'SET_FREQUENCY_BY_FIELD',
+  frequency: e.target.value
+})
+
+const onSetFrequencyBySlider = e => store.dispatch({
+  type: 'SET_FREQUENCY_BY_SLIDER',
+  power: e.target.value
+})
+
+const frequencyToPlaybackRate = (frequency) =>
```
Comment (clarkenciel) [inline src/index.js]: hmm. how can this be written without the implicit `initialFrequency` value?

File: src/index.js (hunk 2 above)
Comment (fakemonster) [reply src/index.js]: yeah I figured I would hold off on that for a bit cause I'm not sure what math we're going to need/not need in the end. though I suppose it wouldn't hurt to just start now

File: src/logo.svg (hunk 3 above)
Comment (fakemonster) [reply src/logo.svg:1]: RIP in peace

**Pull request merged**